src/hotspot/cpu/riscv/globals_riscv.hpp

```diff
@@ -105,6 +105,7 @@ define_pd_global(intx, InlineSmallCode,          1000);
   product(bool, UseZba, false, "Use Zba instructions")                           \
   product(bool, UseZbb, false, "Use Zbb instructions")                           \
   product(bool, UseZbs, false, "Use Zbs instructions")                           \
+  product(bool, UseZcb, false, EXPERIMENTAL, "Use Zcb instructions")             \
```

Comment:
Nit: Can you move this after the line for `UseZfh`. I would like to group EXPERIMENTAL options together.